apps/functions/applications-migration/project-updates-migration/src/execute-sequelize-query.js

```diff
@@ -0,0 +1,22 @@
+import { loadWordpressConfig } from '../../common/config.js';
+import { QueryTypes, Sequelize } from 'sequelize';
+
+/**
+ *
+ * @param {string} query
+ * @param {string[]} replacements
+ */
+export const executeSequelizeQuery = async (query, replacements) => {
+	const config = loadWordpressConfig();
+	const { username, password, database, host, port, dialect } = config.wordpressDatabase;
```

Comment:
DB should be a paramter for the function right?  So for the Welsh one we can pass `config.wordpressDatabaseWelsh`. So we should also add that to the config in this PR. And also do the tf changes first, what do you think?

Reply:
Addressed in latest commit and will be merging the infra PR(https://github.com/Planning-Inspectorate/infrastructure-environments/pull/861) before this one